runtimerevolution/web3-roulette
# [WEB3-24] Participants management page

# Description

This PR adds the participant management page, for admins to approve or reject participants that are in a pending state.

Ticket: https://runtime-revolution.atlassian.net/browse/WEB3-24

## Type of change

- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

1. Create a new giveaway
2. Add a location requirement
3. Add new participants
4. Navigate to the details page of the giveaway
5. Verify that the alert button shows up
6. Click on the button
7. Verify the list of pending participants
8. Approve or reject some of them
9. Verify the new state of the participants
10. Go back and verify the updated number of participants

## Checklist:
